alfalfa_worker/step_sim/model_advancer.py

```diff
@@ -78,9 +78,12 @@ def run(self):
         self.set_idle_state()
         self.init_sim()
         self.set_db_status_running()
+        self.ac.redis_pubsub.subscribe(self.site_id)
         if self.step_sim_type == 'timescale' or self.step_sim_type == 'realtime':
+            self.model_logger.logger.info("Running timescale / realtime")
```

Comment:
@kbenne @corymosiman12  maybe outside scope for now, but we probably want most logging to be "debug" and to have a configurable option to run alfalfa in debug mode.  At least, i don't think that's configurable now.

Reply:
that makes sense

Reply:
tracking via https://github.com/NREL/alfalfa/issues/126